app/containers/AudioPlayer/index.tsx

```diff
@@ -134,16 +136,21 @@ const AudioPlayer = ({
 	useEffect(() => {
 		const unsubscribeFocus = navigation.addListener('focus', () => {
 			audioPlayer.setOnPlaybackStatusUpdate(audioUri.current, onPlaybackStatusUpdate);
+			audioPlayer.addAudioRendered(audioUri.current);
+		});
+		const unsubscribeBlur = navigation.addListener('blur', () => {
+			audioPlayer.removeAudioRendered(audioUri.current);
 		});
 
 		return () => {
 			unsubscribeFocus();
+			unsubscribeBlur();
 		};
 	}, [navigation]);
 
 	useEffect(() => {
 		const listener = EventEmitter.addEventListener(AUDIO_FOCUSED, ({ audioFocused }: { audioFocused: string }) => {
-			setFocused(audioFocused === audioUri.current);
+			setFocused(!!audioFocused && audioFocused === audioUri.current);
```

Comment:
This fixes a visual bug created by the focused state when the audio isn't downloaded yet.